serverless/db/mongo/connection.js

```diff
@@ -10,7 +10,7 @@ const {
   LAMBDA_MONGO_DEV_URI,
   LAMBDA_MONGO_PROD_URI,
 } = process.env;
```

Comment:
let's delete lines 10 - 12.  There's no need for them anymore with your line 13.